blockchain/blockchain-wallet-v4-frontend
# refactor/redux-toolkit-alerts

## Description
Refactor alerts to to use redux-toolkit.

## Testing Steps 
- After entering password and click Sign In, should see alert to check your email.
- Dismiss the alerts by clicking on the 'x' on the alert.
